io.github.wanglin2.mind-map.yml

```diff
@@ -0,0 +1,41 @@
+id: io.github.wanglin2.mind-map
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: run.sh
+rename-icon: thoughts
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+
+modules:
+  - name: mind-map
+    buildsystem: simple
+    build-commands:
+      - ar x thoughts.deb
+      - tar xf data.tar.xz
+      - mv opt/* /app/main
+      - mv usr/share/icons /app/share
+      - install -Dm755 -t /app/bin run.sh
+      - install -Dm644 -t /app/share/applications io.github.wanglin2.mind-map.desktop
+    sources:
+      - type: file
+        only-arches: [x86_64]
+        url: https://github.com/wanglin2/mind-map/releases/download/electron-0.12.1/thoughts_0.12.1_amd64.deb
+        sha256: 96e980ca1766834e901535a77005ef51b94e7c27677ce546e6b647edbf5b9984
+        dest-filename: thoughts.deb
+      - type: file
+        only-arches: [aarch64]
+        url: https://github.com/wanglin2/mind-map/releases/download/electron-0.12.1/thoughts_0.12.1_arm64.deb
+        sha256: c682f466769794dfdf88ff4a720f16fc3e54e1598a4c515955e9436e23694de5
+        dest-filename: thoughts.deb
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - zypak-wrapper.sh /app/main/thoughts "$@"
+      - type: file
+        path: io.github.wanglin2.mind-map.desktop
```

Comment:
why is a .destkop file here? Doesn't the .deb include one? (it's a bug if they don't)